phpunit.xml.dist

```diff
@@ -12,6 +12,9 @@
      syntaxCheck                 = "false">
     <php>
         <ini name="xdebug.max_nesting_level" value="200" />
+        <!-- Disable E_USER_DEPRECATED until reaching compatibility with Symfony 3.0 -->
+        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
+        <ini name="error_reporting" value="-16385"/>
```

Comment:
a better solution is to add a dev dependency on ``symfony/phpunit-bridge`` and setting the appropriate env variable on Travis so that deprecation warnings are not treated as failures